api/app/responses/tests.py

```diff
@@ -985,3 +991,172 @@ def test_response_detail_admin_only(self):
+        question2_2 = self.add_question(application_form.id, section2.id)
+        self.add_question_translation(question2_2.id, 'en', headline='Question 2, S2')
+        question2_3 = self.add_question(application_form.id, section2.id)
+        self.add_question_translation(question2_3.id, 'en', headline='Question 3, S2')
+
+        # Section 3, 1 question
+        section3 = self.add_section(application_form.id)
+        self.add_section_translation(section3.id, 'en', name='Section3')
+        question3_1 = self.add_question(application_form.id, section3.id)
+        self.add_question_translation(question3_1.id, 'en', headline='Queston 1, S3')
+
+        # Add a question that won't have the file attached
+        question_supp2 = self.add_question(application_form.id, section1.id, question_type = 'file')
+        self.add_question_translation(question_supp2.id, 'en', headline='Upload references')
+
+        # Create response
+        self.response1 = self.add_response(application_form.id, self.user1.id, is_submitted=True)
+        self.response1_submitted = self.response1.submitted_timestamp
+        self.response1_started = self.response1.started_timestamp
+        self.response1_id = self.response1.id
+        self.add_answer(self.response1.id, question1.id, 'Section 1 Answer 1')
+        self.add_answer(self.response1.id, question2.id, 'Section 1 Answer 2')
+
+        self.add_answer(self.response1.id, question2_1.id, 'Section 2 Answer 1')
+        self.add_answer(self.response1.id, question2_2.id, 'Section 2 Answer 2')
+        self.add_answer(self.response1.id, question2_3.id, 'Section 2 Answer 3')
+
+        # Add file type answer
+        with tempfile.NamedTemporaryFile(mode='wb') as temp:
+
+            temp.write(b'This is my CV')
+
+            temp.flush()
+
+            buf = BytesIO(open(temp.name, 'rb').read())
+        
+            response_reference = self.app.post(
+                '/api/v1/file',
+                data={'file': (buf, 'test_file')},
+                content_type='multipart/form-data',
+                headers=self.get_auth_header_for('user1@mail.com')
+            )
+
+            file_id = json.loads(response_reference.data)['file_id']
+
+        self.add_answer(
+            self.response1_id,
+            question_supp1_id,
+            json.dumps({"filename": file_id, "rename": "supplementarypdfONE.pdf"})
+        )
+
+
+    def test_zipped_file_uncorrupted(self):
+        """
+        Tests that the zipped files' CRCs are okay. 
+        """
+
+        self._data_seed_static()
+
+        params = {
+            'response_id': self.response1_id,
+            'language': 'en'
+        }
+        
+        response = self.app.get(
+            '/api/v1/response-export',
+            headers=self.get_auth_header_for('event1admin@mail.com'), 
+            json=params)
+            
+        assert response.mimetype == 'application/zip'
```

Comment:
Please use self.assertEqual()  wherever you've used the Python assert statement. The advantage is that assertEqual gives you better error messages when things don't match.